reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-73.md

```diff
@@ -50,7 +43,7 @@ PowerShell 7.3-preview.8 includes the following features, updates, and breaking
 - Add type inference support for generic methods with type parameters (#16951) (Thanks @MartinGC94!)
 - Improve type inference and completions (#16963) (Thanks @MartinGC94!)
   - Allows methods to be shown in completion results for `ForEach-Object -MemberName`
-  - Prevents completion on expressions that return void like `([void](""))`
+  - Prevents completion on expressions that return void like `([void][""])`
```

Comment:
I think this might've come from the link reorganizer.

Reply:
Yep, fixed.